bin/idam-create-caseworker.sh

```diff
@@ -38,6 +38,6 @@ done
 rolesJson="${rolesJson}]"
 
 curl -XPOST \
-  http://localhost:4501/testing-support/accounts \
+  http://localhost:5000/testing-support/accounts \
```

Comment:
port 5000 was used for DB, is it not possible to use another port for SIDAM?

Reply:
I struggled to find the port configuration for this, but really couldn't find it concluding that it is coming from inside the code or something (I wouldn't normally expect).
I decided that we follow the default, the reason being the SIDAM is a common app suite all other teams will also use. It would be worse to change a port of a common service rather than that of our own db.

Reply:
ok fine